components/Notifications/Notification.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function Notification() {
+  return (
+    <div className='absolute right-0 mt-2 bg-white text-gray-800 rounded-lg shadow-lg p-4'>
+      <p>Notification 1</p>
+      <p>Notification 2</p>
+      <p>Notification 3</p>
+    </div>
+  )
+}
```

Comment:
this should take a notification object as prop

Reply:
and should be just 1 of the `p`s

Reply:
the notification needs the following fields:
- opened
- profile picture
- message
- timestamp
- picture

as per the figma design